tests/integration/test_ols.py

```diff
@@ -61,14 +61,15 @@ def test_post_question_on_unexpected_payload() -> None:
     """Check the REST API /v1/query with POST HTTP method when unexpected payload is posted."""
     response = client.post("/v1/query", json="this is really not proper payload")
     assert response.status_code == requests.codes.unprocessable
+    print(response.json())
     assert response.json() == {
         "detail": [
             {
                 "input": "this is really not proper payload",
                 "loc": ["body"],
                 "msg": "Input should be a valid dictionary or object to extract fields from",
                 "type": "model_attributes_type",
-                "url": "https://errors.pydantic.dev/2.6/v/model_attributes_type",
+                "url": "https://errors.pydantic.dev/2.5/v/model_attributes_type",
```

Comment:
we just pinned to v2.6, you may need to rebase

Reply:
Done